apps/web/app/[locale]/page-component.tsx

```diff
@@ -65,69 +65,56 @@ function MainPage() {
-											)}
-										>
-											<div className="mx-8-container pt-9 !px-0 flex flex-row items-start justify-between ">
-												<div className="flex justify-center items-center gap-8 h-10">
-													<PeoplesIcon className="text-dark dark:text-[#6b7280] h-6 w-6" />
-													<Breadcrumb paths={breadcrumb} className="text-sm" />
-												</div>
-												<div className="flex h-10 w-max items-center justify-center   gap-1">
-													<HeaderTabs linkAll={false} />
-												</div>
+				{/* <div className="flex-grow "> */}
+				<MainLayout className="h-full" footerClassName={clsxm('')}>
+					<ChatwootWidget />
+					<div className=" h-full">
+						<ResizablePanelGroup direction="vertical">
+							{/* <Container className="mx-0 " fullWidth={fullWidth}> */}
+							<ResizablePanel
+								defaultSize={50}
+								maxSize={48}
+								className={clsxm(
+									headerSize < 20 ? '!overflow-hidden' : '!overflow-visible',
+									'dark:bg-dark-high border-b-[0.125rem] dark:border-[#26272C]'
+								)}
+								onResize={(size) => setHeaderSize(size)}
+							>
+								<div className="bg-white sticky z-50 dark:bg-dark-high">
+									<div className={clsxm('bg-white dark:bg-dark-high ', !fullWidth && 'x-container')}>
+										<div className="mx-8-container pt-9 !px-0 flex flex-row items-start justify-between ">
+											<div className="flex justify-center items-center gap-8 h-10">
+												<PeoplesIcon className="text-dark dark:text-[#6b7280] h-6 w-6" />
+												<Breadcrumb paths={breadcrumb} className="text-sm" />
 											</div>
-											<div className="mx-8-container mb-1">
-												<UnverifiedEmail />
-												<TeamInvitations />
-												<TeamOutstandingNotifications />
-												{isTeamMember ? (
-													<TaskTimerSection isTrackingEnabled={isTrackingEnabled} />
-												) : null}
+											<div className="flex h-10 w-max items-center justify-center  gap-1">
+												<HeaderTabs linkAll={false} />
 											</div>
-											<TeamMemberHeader view={view} />
 										</div>
+										<div className="mx-8-container mb-1">
+											<UnverifiedEmail />
+											<TeamInvitations />
+											<TeamOutstandingNotifications />
+											{isTeamMember ? (
+												<TaskTimerSection isTrackingEnabled={isTrackingEnabled} />
+											) : null}
+										</div>
+										<TeamMemberHeader view={view} />
 									</div>
-								</ResizablePanel>
-								<ResizableHandle withHandle />
-
-								{/* </Container> */}
-								<ResizablePanel
-									defaultSize={55}
-									maxSize={95}
-									className="!overflow-y-scroll custom-scrollbar"
-								>
-									<div>{isTeamMember ? <TeamMembers kanbanView={view} /> : <NoTeam />}</div>
-								</ResizablePanel>
-							</ResizablePanelGroup>
-						</div>
-					</MainLayout>
-				</div>
-				<div className="bg-white dark:bg-transparent ">
-					<Divider />
-					<Footer
```

Comment:
@CREDO23 footer removed, why!?

Reply:
It was a duplicate !
There is a Footer in the `<MainLayout/>` component.